lib/wipe-dependencies.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
```

Comment:
I added the comments.... It's basically a helper function to update all dependencies as an optional task...